nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -162,16 +163,26 @@ def repos_from_config_file(config_file)
                     node.at_xpath("./value")&.content&.strip
               }
             end
```

Comment:
You can delete this whole sources block since we're handling that below.

Reply:
Completed. Good catch. Thanks!